samples/geolocation-weather/src/helpers.ts

```diff
@@ -0,0 +1,31 @@
+const getClientAddressFromRequest = (req: Request): string | null => {
```

Comment:
@ThorstenHans I borrowed this from one of your js samples. It'd be great to figure out how to consolidate this logic to be used across multiple sample apps

Reply:
Hmm, what are you thinking about? Sharing this across multiple apps/examples would either lead to a local or published npm package. 

IMO, being repetitive here is okay, for the sake of each sample being holistic.

Or had you something different in mind @michelleN

Reply:
yea, npm package was along the lines of what I was thinking but good point on being repetitive for the sake of being holistic. I think we should keep it as is.